go.mod

```diff
@@ -0,0 +1,9 @@
+module github.com/ebay/libovsdb
+
+go 1.12
```

Comment:
According to https://github.com/golang/go/wiki/Modules#gomod there is no "go" directive in go.mod. Please correct me if I misunderstood anything. I am new to "go mod". Is this file generated by go command or did you update it manually?

Reply:
this is generated automatic by go mod init github.com/ebay/libovsdb
and started from go 1.12 this string automatic added to go.mod file